src/ReleaseHistory.md

```diff
@@ -5,6 +5,7 @@
 * FEATURE: Add new PE `CV_CFL_LANG` language code for `ALIASOBJ` and `Rust`. [530](https://github.com/microsoft/binskim/pull/530)
 * BUGFIX: Fix `BA2014.DoNotDisableStackProtectionForFunctions` to eliminate false positive reports that `GsDriverEntry` has disabled the stack protector. [551](https://github.com/microsoft/binskim/pull/551)
 * BREAKING: Rename `BA2026.EnableAdditionalSdlSecurityChecks` to `BA2026.EnableMicrosoftCompilerSdlSwitch` to clarify rule purpose. [#586](https://github.com/microsoft/binskim/pull/586)
+* BUGFIX: Fix inability to emit `SummaryEvent` when SARIF V1. [613](https://github.com/microsoft/binskim/pull/613)
```

Comment:
>Fix [](http://example.com/codeflow?start=10&length=3)

Make this release note searchable based on what customers see when this crashes.